lib/front_end.rb

```diff
@@ -3,7 +3,7 @@ def self.host
     if Rails.env.production?
       'https://www.zooniverse.org'
     else
-      'http://demo.zooniverse.org/panoptes-front-end'
+      'https://pfe-preview.zooniverse.org'
```

Comment:
I'm going to leave that as-is for now. Easy enough for someone to change it if it's ever needed.